### ISSUE TITLE
RevEng: Namespace is missing first character

### ISSUE THREAD

**Issue by bricelam** [aspnet/EntityFramework#3228]
The following command...
```ps1
Scaffold-DbContext -OutputDirectory Models 'Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=Northwind;Integrated Security=True' EntityFramework.SqlServer
```
...outputs code with the following namespace.
```C#
namespace ConsoleApplication1.odels
{
    // ...
}
```

**Comment by lajones** [comment]
Fixed by PR #3447.

**Comment by julielerman** [comment]
ahh and another one that you already know about. :)
